xbmc/video/Teletext.cpp

```diff
@@ -1337,7 +1337,10 @@ void CTeletextDecoder::DoFlashing(int startrow)
   /* Flashing */
   TextPageAttr_t flashattr;
   char flashchar;
-  long flashphase = std::chrono::steady_clock::now().time_since_epoch().count() % 1000;
+  long flashphase = std::chrono::duration_cast<std::chrono::milliseconds>(
```

Comment:
Could you change flashphase to be `std::chrono::milliseconds`. Then you can drop the `.count()` and use `% 1000ms`.

That way we can update the existing comparisons with the `ms` literal. That way we know we are comparing milliseconds.

Reply:
done